src/ucm/cuda/cudamem.c

```diff
@@ -228,6 +257,17 @@ static ucm_cuda_func_t ucm_cuda_driver_funcs[] = {
     {{NULL}, NULL}
 };
 
+static size_t ucm_cuda_get_symbol_size(const void *symbol)
+{
+    size_t size;
+
+    if (cudaGetSymbolSize(&size, symbol) != cudaSuccess) {
```

Comment:
should it be?
```suggestion
    if (ucm_orig_cudaGetSymbolSize(&size, symbol) != cudaSuccess) {
```

Reply:
No, we do not change `cudaGetSymbolSize`. Thus, `ucm_orig_cudaGetSymbolSize` is not defined.